specifications/iati-register-your-data-api.yaml

```diff
@@ -0,0 +1,1413 @@
+        user_id:
+          type: string
+          format: uuid
+          example: 6423b556-73de-4969-a2f1-91a38bca92b3
+        created_date:
+          type: string
+          format: date-time
+          example: "2023-12-25T23:00:12+00:00"
+    DatasetAction:
+      description: Example of a Dataset Action where a dataset has been updated.
+      type: object
+      properties:
+        action_type:
+          type: string
+          enum:
+            - create
+            - update_metadata
+            - update_url
+            - change_visibility
+            - delete
+          example: update_url
+        user_application_name:
+          type: string
+          example: IATI Account
+        user_application_id:
+          type: string
+          format: uuid
+          example: 59530216-574b-4b68-8489-ca869fcee193
+        user_name:
+          type: string
+          example: John Smith
+        user_id:
+          type: string
+          format: uuid
+          example: 6423b556-73de-4969-a2f1-91a38bca92b3
+        created_date:
+          type: string
+          format: date-time
+          example: "2023-12-25T23:00:12+00:00"
+    #
+    # Reporting org response schemas
+    #
+    ReportingOrgList:
+      description: Schema for a list of reporting organisations.
+      properties:
+        status:
+          type: string
+          example: success
+        error:
+          type: object
+          example: null
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                format: uuid
+                example: 5401c7f4-e9fd-4ce3-b184-bfe1cb71a9f8
+              user_role:
+                type: string
+                example: editor
+              metadata:
+                allOf:
+                  - $ref: "#/components/schemas/ReportingOrgReadableData"
+              reporting_org_actions:
+                type: array
+                items:
+                  $ref: "#/components/schemas/OrgAction"
+    ReportingOrgCreated:
+      description: Example of the response where an organisation has been created.
+      properties:
+        status:
+          type: string
+          example: success
+        error:
+          type: object
+          example: null
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                format: uuid
+                example: 5401c7f4-e9fd-4ce3-b184-bfe1cb71a9f8
+              user_role:
+                type: string
+                example: admin
+              metadata:
+                allOf:
+                  - $ref: "#/components/schemas/ReportingOrgReadableData"
+                  - type: object
+                    properties:
+                      registry_approved:
+                        example: false
+                      first_publication_date:
```

Comment:
Should these be removed as they are included in the `ReportingOrgReadableData` schema?